### ISSUE TITLE
Code Table Request - identifiers - HerpMapper ID

### ISSUE THREAD

**Issue by dustymc** [ArctosDB/arctos#6439]
## Initial Request

Convert HerpMapper ID including base URL to type identifier

Data:
[temp_herpmapper_id.csv.zip](https://github.com/ArctosDB/arctos/files/11837357/temp_herpmapper_id.csv.zip)

Summary:

```

 guid_prefix | numrecs 
-------------+---------
 APSU:Herp   |       1
 UCM:Herp    |      29

```

 @ebraker
 @gradyjt

@ArctosDB/arctos-code-table-administrators 

## Approval

_All of the following must be checked before this may proceed._

_The [How-To Document](https://handbook.arctosdb.org/how_to/How-To-Manage-Code-Table-Requests.html) should be followed. Pay particular attention to terminology (with emphasis on consistency) and documentation (with emphasis on functionality). **No person should act in multiple roles**; the submitter cannot also serve as a Code Table Administrator, for example._

- [x] Code Table Administrator[1] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] Code Table Administrator[2] - check and initial, comment, or thumbs-up to indicate that the request complies with the how-to documentation and has your approval
- [x] DBA - The request is functionally acceptable. The term is not a functional duplicate, and is compatible with existing data and code.
- [x] DBA - Appropriate code or handlers are in place as necessary. (ID_References, Media Relationships, Encumbrances, etc. require particular attention)

## Rejection

_If you believe this request should not proceed, explain why here. Suggest any changes that would make the change acceptable, alternate (usually existing) paths to the same goals, etc._

1. _Can a suitable solution be found here? If not, proceed to (2)_
2. _Can a suitable solution be found by Code Table Committee discussion? If not, proceed to (3)_
3. _Take the discussion to a monthly Arctos Working Group meeting for final resolution._

## Implementation

_Once all of the Approval Checklist is appropriately checked and there are no Rejection comments, or in special circumstances by decree of the Arctos Working Group, the change may be made._

_Review everything one last time.  Ensure the How-To has been followed. Ensure all checks have been made by appropriate personnel._

_Make changes as described above. Ensure the URL of this Issue is included in the definition._

_Close this Issue._

_**DO NOT** modify Arctos Authorities in any way before all points in this Issue have been fully addressed; data loss may result._

## Special Exemptions

_In very specific cases and by prior approval of The Committee, the approval process may be skipped, and implementation requirements may be slightly altered. Please note here if you are proceeding under one of these use cases._

1. _Adding an existing term to additional collection types may proceed immediately and without discussion, but doing so may also subject users to future cleanup efforts. If time allows, please review the term and definition as part of this step._
2. _The Committee may grant special access on particular tables to particular users. This should be exercised with great caution only after several smooth test cases, and generally limited to "taxonomy-like" data such as International Commission on Stratigraphy terminology._

**Comment by Jegelewicz** [comment]
This crept into my brain while I was on vacation. Instead of "identifier" would it make more sense to have the type be "url"? The path to finding all of the things with webpages?

url - A Uniform Resource Locator (URL) that is a reference to a web resource about or related to the catalog record.

**Comment by dustymc** [comment]
> would it make more sense to have the type be "url"? 

Why force unnecessary pigeonholes on anyone? I can see no reason to do anything beyond this decision tree:

* Does it DO STUFF in some local system which exists for some compelling reason?
    * YES ---> use NK, AF, etc. - there's no mystery for the users here
    * NO ---> use identifier, still no mystery

**Comment by ebraker** [comment]
@dustymc for all these new requests, can you clarify what is happening? Just 'other ID' becomes 'Identifier' (semantics)? Does anything change on the user end, i.e. will we still have the term available on a drop down menu?

**Comment by dustymc** [comment]
@ebraker https://github.com/ArctosDB/arctos/issues/6439#issuecomment-1603042326

The current decision tree branches out to a bunch of types from NO, and from there to how to handle those types, (this one says "(format is integer only, exclude "HM" prefix)." in the documentation, which I suspect is even more evidence of the blurry line between identifiers and other stuff in the current model) which leads to nonsense data (there are some convenient examples in https://github.com/ArctosDB/arctos/issues/6443).

So when you find https://www.herpmapper.org/record/323313 instead of going through all that nonsense, you just pick 'identifier' (which should be a lot more obvious once we get through some of these Issues) and paste in ``https://www.herpmapper.org/record/323313`` (eg from your browser) and everything works, in Arctos and beyond.

(And the type isn't necessary, if we really want to simplify....)
